pages/price-feeds/best-practices.mdx

```diff
@@ -70,6 +73,18 @@ This analogy suggests two simple solutions to races:
    The protocol can require a short delay between the two transactions, and the user's request gets fulfilled at the Pyth price as of the second transaction.
    This technique gives the protocol extra time to observe price changes, giving it a head start in the latency race.
 
+### Latency Mitigations for Derivative Protocols
+
+To mitigate the risk of latency, derivative protocols should consider the following strategies:
```

Comment:
I think it's good if you emphasize why even the slightest latency can affect derivatives. (leveraged markets)

Also, "derivative protocols are encouraged to consider the following strategies"

Reply:
It was mentioned above in the latency section.